src/components/TokenImg.tsx

```diff
@@ -39,7 +45,13 @@ function TokenImg({ token, ...rest }: TokenImgProps) {
     [src]
   )
 
-  if (!src) return <MissingTokenImg src={missingTokenSrc} alt={alt} color="secondary" {...rest} />
+  if (!src) {
+    return (
+      <MissingTokenImg>
```

Comment:
This should continue to pass `{...rest}`, as these are the DOM attributes being passed down.